lib/cc/engine/analyzable_files.rb

```diff
@@ -27,13 +26,17 @@ def fetch_files(paths)
       end
 
       def build_files_with_inclusions(inclusions)
-        fetch_files(inclusions)
+        if inclusions == ["./"]
```

Comment:
I guess my suspicion is there's a way to fix this by only changing `#fetch_files`. Do you think that's true?

- `#{path}/` is a clear but, so let's assume that's fixed
- The remaining bug can be restated as:

`#fetch_files` returns results with `path` pre-pended, but that doesn't match the paths returned by the analysis. Can you unpack in what way they mismatch?

Are the analysis results relative to (each) `path`, or relative to the top-level `code` directory?